spec/01-unit/05-utils_spec.lua

```diff
@@ -197,6 +197,16 @@ describe("Utils", function()
       assert.False(validate_utf8(string.char(255))) -- impossible byte
       assert.False(validate_utf8(string.char(237, 160, 128))) -- Single UTF-16 surrogate
     end)
+
+    it("checks valid nginx size values", function()
+      local parse_ngx_size = require("kong.tools.string").parse_ngx_size
+
+      assert.equal(1024 * 1024 * 1024, parse_ngx_size("1G"))
```

Comment:
Should we add test case for `g` and `m`?

Reply:
added